src/sql/azdata.proposed.d.ts

```diff
@@ -517,6 +517,20 @@ declare module 'azdata' {
 		export function getNonDefaultOptions(profile: IConnectionProfile): Thenable<string>;
 	}
 
+	export interface ConnectionInfoSummary {
+		/**
+		 * Process ID returned from service host, if connection was successful.
+		 */
+		pid?: string | undefined;
```

Comment:
What about `serverConnectionId`? "Process" is almost never going to be accurate (despite the name that SPID has...) - especially for other providers. serverConnectionId is much more generic and descriptive of what this value is intended to be (a unique value identifying the connection on the server)

@erinstellato-ms Do you have any preference here?

At the very least let's fully spell this out as `processId`, not reason to shorten it like this IMO

Reply:
I noted previously that this has been known as session_id within SQL Server since the SQL 2005 release (when Dynamic Management Views (DMVs)) were introduced.  Prior to that, it was referred to as SPID (and still is in very old system views like sysprocesses).

I do not support pid, or even processID, because that's Windows specific - every process that runs in Windows has a process ID (PID).  Using that here seems very wrong to me.

In Oracle is it known as Session ID (SID).
In MySQL it seems to be conn_id (and pid is client process ID)
In PostgreSQL it is...process_id

I would argue for session_id, but I'm also not clear on where we are displaying this text.

Reply:
Sounds good, we can probably do a separate follow up for the internal naming if we end up wanting to change that.